mkosi/distributions/gentoo.py

```diff
@@ -16,56 +17,91 @@
 from mkosi.remove import unlink_try_hard
 from mkosi.run import run, run_workspace_command
 from mkosi.state import MkosiState
+from mkosi.types import PathString
+
+thread_counts = (os.cpu_count() or 1) * 2
```

Comment:
Please move this variable to invoke_emerge(). Also, why multiply by 2?

Reply:
2 is for hypethreading.